backend/pokemon/tests.py

```diff
@@ -79,3 +82,69 @@ def test_if_team_is_valid(self):
 
         self.assertTrue(sum_pokemon_team_1)
         self.assertFalse(sum_pokemon_team_2)
+
+
+class PokemonApiIntegrationTest(TestCaseUtils):
+    def setUp(self):
+        super().setUp()
+        self.opponent = baker.make("users.User")
+        self.battle = baker.make("battling.Battle", creator=self.user, opponent=self.opponent)
+        self.team = baker.make("battling.Team", battle=self.battle, trainer=self.user)
+
+    @patch("pokemon.helpers.get_pokemon_from_api")
+    def test_if_pokemon_api_integration_returns_pokemon_data(self, mock_get_pokemon):
```

Comment:
This test should be on the CreateFormTest class. Since you are testing the form. The name is also a bit misleading